sklearnex/preview/ensemble/_forest.py

```diff
@@ -547,26 +649,15 @@ def _onedal_cpu_supported(self, method_name, *data):
                 [
                     (
                         daal_check_version((2023, "P", 200))
-                        or self._estimator.__class__ == DecisionTreeClassifier,
+                        or self.estimator.__class__ == DecisionTreeClassifier,
```

Comment:
_estimator is set after fit, not before, but estimator attribute is set on instantiation. Chose to use more sklearn-conformant setup.